server/proxy/pf_context.h

```diff
@@ -35,52 +35,45 @@
  * of the connection. This lets one side of the proxy to forward events to the
  * other.
  */
-struct proxy_context
-{
-	/* Context of client or server */
-	rdpContext _context;
-
-	rdpProxyServer* server;
 
-	/**
-	 * Context of peer to which the proxy is connected.
-	 * Events from the first context are forwarded to this one.
-	 */
-	rdpContext* peerContext;
-
-	HANDLE connectionClosed;
-};
-typedef struct proxy_context proxyContext;
+typedef struct proxy_data proxyData;
 
-/**
- * Context used for the client's connection to the proxy.
- */
-struct client_to_proxy_context
+struct pserver_context
```

Comment:
@speidy What do you think, should this be `pserver_context` or `p_server_context`, because the camel case version is `pServerContext`

Reply:
Thing is, if we chose `p_server` instead of `pserver`, the methods involving `p_server` would look weird (ala `init_p_server_context`). Maybe we should stick to `pserver` and alias it as `pserverContext` instead of `pServerContext`, I just think we need to be consistent.

Reply:
I fixed the licenses. About `pserverContext`, I don't like it because it is less understandable than `pServerContext`.

Reply:
pServerContext